modules/nw-configuring-external-load-balancer.adoc

```diff
@@ -29,9 +27,11 @@ to use an external load balancer in place of the default load balancer.
 
 * On your load balancer, port 22623, which is used to serve ignition startup configurations to nodes, is not exposed outside of the cluster.
 
-* Your load balancer must be able to access every machine in your cluster. Methods to allow this access include:
-** Attaching the load balancer to the cluster's machine subnet.
-** Attaching floating IP addresses to machines that use the load balancer.
+* Your load balancer can access the required ports on each node in your cluster. You can ensure this level of access by completing the following actions:
++
+** The API load balancer can access ports `22623` and `6443` on the control plane nodes.
++
+** The ingress load balancer can access ports `443` and `80` on the nodes where the ingress pods are located.
 
 [IMPORTANT]
 ====
```

Comment:
SME query: Should I delete this IMPORTANT notice? 

Feedback on Jira:

> 2.  The below quoted section is confusing:

"""
[IMPORTANT]
====
External load balancing services and the control plane nodes must run on the same L2 network, and on the same VLAN when using VLANs to route traffic between the load balancing services and the control plane nodes.
====

Reply:
yeah, that is a bit confusing.  i think it's fine to omit.

Reply:
otherwise LGTM

Reply:
I raised https://github.com/openshift/openshift-docs/pull/62749 to remove the note for all our versioned docs (4.14 through to 4.10)